tests/utils/StringTest.cpp

```diff
@@ -32,11 +31,27 @@ TEST_P(WriteInt2mmTest, WriteInt2mm)
     ASSERT_TRUE(ss.good()) << "The integer " << in << " was printed as '" << str << "' which was a bad string!";
 
     const int out = MM2INT(strtod(str.c_str(), nullptr));
-    ASSERT_EQ(in, out) << "The integer " << in << " was printed as '" << str << "' which was interpreted as " << out << " rather than " << in << "!";
+    ASSERT_EQ(in, out) << "The integer " << in << " was printed as '" << str << "' which was interpreted as " << out << " rather than "
+                       << in << "!";
 }
 
-INSTANTIATE_TEST_CASE_P(WriteInt2mmTestInstantiation, WriteInt2mmTest,
-        testing::Values(-10000, -1000, -100, -10, -1, 0, 1, 10, 100, 1000, 10000, 123456789, std::numeric_limits<int32_t>::max() / 1001)); //For max integer test, divide by 1000 since MM2INT multiplies by 1000 which would cause an overflow.
+INSTANTIATE_TEST_CASE_P(
+  WriteInt2mmTestInstantiation,
+  WriteInt2mmTest,
+  testing::Values(-10000,
+                  -1000,
+                  -100,
+                  -10,
+                  -1,
+                  0,
+                  1,
+                  10,
+                  100,
+                  1000,
+                  10000,
+                  123456789,
+                  std::numeric_limits<int32_t>::max()
+                    / 1001)); // For max integer test, divide by 1000 since MM2INT multiplies by 1000 which would cause an overflow.
```

Comment:
It's best to keep this division on the same line as the `::max()`, to indicate that it belongs to that one. One might think that we're feeding max int as a test value here.